Sources/StreamChat/Database/DTOs/MessageDTO.swift

```diff
@@ -721,7 +721,7 @@ extension NSManagedObjectContext: MessageDatabaseSession {
         dto.user = user
 
         dto.reactionScores = payload.reactionScores.mapKeys { $0.rawValue }
-        dto.reactionCounts = payload.reactionScores.mapKeys { $0.rawValue }
+        dto.reactionCounts = payload.reactionCounts.mapKeys { $0.rawValue }
```

Comment:
I also have a TODO/fix about this in my openapi branch. Is it just a typo or there was some specific reason, @nuno-vieira do you know maybe?